datamade/just-spaces
# List, detail, and delete views for studies and locations

## Overview

Closes #134. This PR creates list, edit, and delete views for PLDP Study and Location models. This means that users can look at details of studies and locations once they're created, and that we should make sure that only some users have permission to do the deleting once we finalize #48.

## Testing Instructions

* Run migrations
* Navigate to `http://127.0.0.1:8000/surveys/create/`
* In the `Location` and `Study` field you should see links to list and detail views for each. Click around and make sure all behave as you'd expect, and that you can successfully deactivate both once created.
